composer.json

```diff
@@ -30,6 +30,7 @@
         "symfony/polyfill-php73": "self.version",
         "symfony/polyfill-php74": "self.version",
         "symfony/polyfill-php80": "self.version",
+        "symfony/polyfill-php81": "self.version",
```

Comment:
this is broken because you forgot to update the autoloading config of this root package to make it load the new code.

Reply:
apparently, we also missed loading the php80 bootstrap file here?
fixed!

Reply:
PHP 8 is loaded in `src/bootstrap.php` due to the different PHP requirement